app/middlewares/auth.js

```diff
@@ -0,0 +1,21 @@
+const sessionManager = require('./../services/sessionManager'),
+  User = require('../models').User;
+
+exports.secure = (req, res, next) => {
+  const auth = req.headers[sessionManager.HEADER_NAME];
+
+  if (auth) {
+    const email = sessionManager.decode(auth);
```

Comment:
in particular you here are decoding the payload of the token. Which in this case is `{ email: some@email.com }`, it will get confusing if the payload grows because you will have assigning to the email var something with more attributes than the email.
If i wasn't clear enough let me know and we talk about it.